backend/dataall/core/stacks/__init__.py

```diff
@@ -1 +1 @@
-from dataall.core.stacks import api
+from dataall.core.stacks import api, handlers
```

Comment:
shouldn't we include here `tasks` too?

Reply:
It is not necessary. The tasks are references by the fullname